homeassistant/components/zwave_me/light.py

```diff
@@ -57,8 +75,17 @@ def turn_on(self, **kwargs: Any) -> None:
         color = kwargs.get(ATTR_RGB_COLOR)
 
         if color is None:
-            color = (122, 122, 122)
-        cmd = "exact?red={}&green={}&blue={}".format(*color)
+            brightness = kwargs.get(ATTR_BRIGHTNESS)
+            if brightness is None:
+                self.controller.zwave_api.send_command(self.device.id, "on")
+            else:
+                self.controller.zwave_api.send_command(
+                    self.device.id, f"exact?level={round(brightness / 2.55)}"
+                )
+            return
```

Comment:
Why don't we allow the user to change brightness and color in the same service call?

Reply:
Brightness is calculated from current color values, changing both simultaneously can lead to unexpected behavior, such as wrong color being selected

Reply:
How is brightness calculated if the light doesn't support rgb color mode?

Reply:
In that case only single value - brightness is provided instead of RGB values

Reply:
Ok.